src/Turn.js

```diff
@@ -0,0 +1,31 @@
+class Turn {
+  constructor(guess, cardInPlay) {
+    this.guess = guess;
+    this.cardInPlay = cardInPlay;
+  }
+
+  returnGuess() {
+    return this.guess;
+  }
+
+  returnCard() {
+    return this.cardInPlay;
+  }
+
+  evaluateGuess() {
+    if (this.guess === this.cardInPlay.correctAnswer) {
+      return true;
+    }
+    return false;
+  }
+
+  giveFeedback() {
+    var guessIsCorrect = this.evaluateGuess();
```

Comment:
I created a new variable here in order to avoid nested if statements. This could potentially be refactored to be more concise.